keras_retinanet/models/mobilenet.py

```diff
@@ -0,0 +1,62 @@
+import keras
```

Comment:
Thanks for the PR!

Could you add a license and copyright notice to the top of the file?

Reply:
done